test/quantization/core/test_quantized_tensor.py

```diff
@@ -813,8 +813,13 @@ def test_clone(self):
                 [numel], scale=scale, zero_point=zero_point,
                 device=device, dtype=dtype)
             per_channel_quantized = torch._empty_per_channel_affine_quantized(
-                [numel], scales=torch.tensor([scale]), zero_points=torch.tensor([zero_point]), axis=0,
-                device=device, dtype=dtype)
+                [numel],
```

Comment:
What's the change here?

Reply:
The number of elements of `scales` and `zero_points` did not match the values. Before we only compared the values so this worked out. Now we compare the `.dequantize`'d version.